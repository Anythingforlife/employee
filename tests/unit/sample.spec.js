import { shallowMount } from "@vue/test-utils";
import Sample from "@/components/sample.vue";

describe("Sample.vue", () => {

  let wrapper = null;

  beforeEach(() => {
    wrapper = shallowMount(Sample, {
      propsData: {
        title: 'sample App'
      }
    })
  })

  afterEach(() => {
    wrapper.destroy();
  })

  it('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it("renders props.title when passed", () => {
    expect(wrapper.text()).toMatch('sample App');
  });

  test('adds 1 + 2 to equal 3', () => {
    expect(wrapper.vm.sum(1, 2)).toBe(3);
  });


});