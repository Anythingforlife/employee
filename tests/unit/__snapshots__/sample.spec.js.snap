// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample.vue renders correctly 1`] = `
<h1>
  
</h1>
`;
